src/fides/api/schemas/connection_configuration/connection_secrets_rds_mysql.py

```diff
@@ -1,26 +1,14 @@
-from pydantic import Field
-
 from fides.api.schemas.base_class import NoValidationSchema
-from fides.api.schemas.connection_configuration.connection_secrets_base_aws import (
-    BaseAWSSchema,
+from fides.api.schemas.connection_configuration.connection_secrets_base_rds import (
+    BaseRDSSchema,
 )
 
 
-class RDSMySQLSchema(BaseAWSSchema):
+class RDSMySQLSchema(BaseRDSSchema):
     """
     Schema to validate the secrets needed to connect to a RDS MySQL Database
     """
```

Comment:
do we need to keep the  db-specific schemas, since they don't extend the behavior of BaseRDSSchema in any way?